chapter02/qli/src/main/java/spring/training/model/Account.java

```diff
@@ -0,0 +1,107 @@
+package spring.training.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Account {
+	private int accountNumber;
+	private String accountDescription;
+	private double accountAmount;
+	private Customer customer;
+	
+	public final static double MAX_AMOUNT = 100000000;
+
+	private List<AccountTranscation> transactions = new ArrayList<>();
+	
+	
+	public int getAccountNumber() {
+		return accountNumber;
+	}
+
+	public void setAccountNumber(int accountNumber) {
+		this.accountNumber = accountNumber;
+	}
+
+	public String getAccountDescription() {
+		return accountDescription;
+	}
+
+	public void setAccountDescription(String accountDescription) {
+		this.accountDescription = accountDescription;
+	}
+
+	public double getAccountAmount() {
+		return accountAmount;
+	}
+
+	public void setAccountAmount(double accountAmount) {
+		this.accountAmount = accountAmount;
+	}
+
+	public Customer getCustomer() {
+		return customer;
+	}
+
+	public void setCustomer(Customer customer) {
+		this.customer = customer;
+	}
+
+	public List<AccountTranscation> getTransactions() {
+		return transactions;
+	}
+
+	public boolean recordTransactions(AccountTranscation txn) {
```

Comment:
Hmm.. i missed this part :)
But I feel, it is not the responsibility of account to record it, it may be a part of the responsibility of the Bank.